src/oruguru.c

```diff
@@ -10,15 +36,24 @@ int main(int argc, char *argv[]) {
   SetTargetFPS(60);
 
   while (!WindowShouldClose()) {
+  Vector2 v = {100, 100};
+  Effect effect;
+  spawnEffect(&effect, v);
```

Comment:
These are tabbed incorrectly; also in the finished product, we should try to keep variable declarations above the block they are used in.